RELEASING.md

```diff
@@ -0,0 +1,24 @@
+## Releasing Theme Check Action
+
+1. Check the Semantic Versioning page for info on how to version the new release: [semver.org](http://semver.org).
+
+2. Run [`git changelog`](https://github.com/tj/git-extras) to update `CHANGELOG.md`.
+
+3. Run `export version=1.1.1` to set the version number for the new release (replace 1.1.1 with the version number you're releasing).
+
+4. Run `bin/release v$version` to push a new tag
+    - If anything goes wrong during the release process, you may:
+      - 4.1. Run `bin/rollback-release v$version` to delete a published tag
+      - 4.2. Run `bin/re-release v$version` to re-release the tag
```

Comment:
IMO one of the best things you can do in a procedure like that is to put in "easy to copy and paste" stuff.

So as much as possible, I try to follow the following formula:

1. [Action verb] to [do something] by running the following command:

   ```bash
   # here I put a command without the prefixed $ so that folks can copy i
   some_arg=X.X.X
   command $some_arg
   ```

The rationale being that I don't want to think when I'm doing this and I want to get it done as fast as possible so that I can get back to providing value.